source/introduction.txt

```diff
@@ -0,0 +1,263 @@
+
+To learn more about the Realm GraphQL API, see :ref:`graphql-api`.
+
+.. _intro-web-mql:
+
+Client-Side MongoDB Queries
+```````````````````````````
+
+Traditionally developers query the database on the server and access the server
+from the client through a HTTP API. This requires more developer resources and lacks
+the flexibility of directly querying a database.
+
+{+backend+} includes the :manual:`Query API </tutorial/query-documents>`,
+a powerful API that lets you access {+atlas+} data from the client.
+You can query {+atlas-short+} like you're using a database driver and {+service-short+} enforces 
+:ref:`data access rules <intro-web-schema>` rules for all requests.
+
+To learn more about the built-in MongoDB service, see :ref:`data-sources`.
+
+.. _intro-device-sync:
+
+Synchronize Data Across Devices
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Synchronizing data is a hard problem for mobile app developers. Realm Sync
+provides offline-first sync capabilities for your mobile application. 
+Client applications use :ref:`Realm Database <intro-realm-database>` to 
+persist data to the local device. When the device has a network connection, 
+Sync seamlessly sends the data to the Realm App. The app stores the data 
+to a linked :ref:`MongoDB Atlas data source <data-sources>`, and updates 
+the data on other client devices.
+
+Realm Sync provides :ref:`conflict resolution <conflict-resolution>` to 
+handle multiple users updating the same data. When combined with :ref:`user 
+authentication and management <intro-user-authentication>` and :ref:`schema 
+validation and data access <intro-web-schema>`, Realm Sync provides a secure
+way for your mobile clients to sync data across devices and with the cloud.
+
+Get Started with MongoDB Realm
+------------------------------
+
+You can ship your first app built with MongoDB Realm in minutes.
+
+Tutorials
+~~~~~~~~~
+
+Check out our :doc:`tutorials </tutorials>` to learn more about how to build 
+Realm Applications. Tutorials start with git repositories you can clone
+and run, and walk you through code excerpts that are specific to working
+with Realm.
+
+To get started with your first web app, follow the :ref:`Task Tracker 
+application tutorial <tutorial-task-tracker-web>`. You can also refer to the 
+:ref:`Web SDK <web-intro>` documentation for TypeScript and JavaScript.
+
+To get started with a backend app, follow our :ref:`GitHub Issue Tracker
+tutorial <backend-tutorial>`. 
+
+Prebuilt Applications
+~~~~~~~~~~~~~~~~~~~~~
+
+To explore a prebuilt app that you can customize, create a template app.
+Template applications are working apps that use Realm services to do things 
+like:
+
+- Use Sync to synchronize data between mobile clients and the linked 
+  MongoDB Atlas collection.
+- Host a Todo web app using the :ref:`GraphQL API <graphql-api>`
+- Host a Todo web app using the :ref:`Realm Web SDK <web-intro>`
```

Comment:
```suggestion
- Host a Todo web app using the :ref:`Realm Web SDK <web-intro>`
- Create a Todo mobile application using the `Realm client SDKs <https://www.mongodb.com/docs/realm/>`__
```

curiously there's no ref for the Realm SDKs homepage, so providing URL link to it here. maybe there's a better way tho..